### ISSUE TITLE
Create CEP-78 JS Client Tutorial

### ISSUE THREAD

**Issue by devendran-m** [casper-network/docs#615]
This is the base code to be used for the tutorial: https://github.com/casper-ecosystem/cep-78-enhanced-nft/tree/js-client/client-js.
The main requirement here is to add a TUTORIAL.md in the repository above, explaining different aspects of the code and tests already found in the repository. 
Then, we need to update the landing page to link to this tutorial from here: https://docs.casperlabs.io/dapp-dev-guide/building-dapps/sdk/script-sdk/#repository-7-client-packages.

**Acceptance criteria**:
- understand the new code and tests (2 points)
- update the landing page and any links to the old CEP-47 tutorial in the main docs (2 point)
- write the tutorial (2 points)
- send the tutorial for review and incorporate feedback (2 points)
Total: 8 points

**Comment by ACStoneCL** [comment]
For reference: https://docs.casperlabs.io/dapp-dev-guide/building-dapps/sdk/script-sdk/#repository-7-client-packages

This document will need to be updated when the JS Client becomes available for CEP-78.

**Comment by RitaMAllenCA** [comment]
@ipopescu need an estimate, please

**Comment by ipopescu** [comment]
@RitaMAllenCA I will check what exactly we need here and get back to you.

**Comment by ipopescu** [comment]
@RitaMAllenCA I cannot assign points for some reason, but I have estimated it to be 8 points. I added a detailed description with subtasks (the description was previously empty).
Second, I spoke with Jan, and the developers need to do some more work before we can release the new cep-78 contract and example client:
- fix gas cost 
- update the tests for the JS client
- add basic comments and a readme with basic usage of the JS client
This work is estimated to start this sprint. So, I recommend adding this card to the beginning of the next sprint (mid-Nov).
